build/package/powershell/windows_post_package_test.ps1

```diff
@@ -0,0 +1,21 @@
+# Install current version of MongoDB Atlas CLI
```

Comment:
[q] can we add lines in between the comment blocks?

Reply:
Of course, I divided the script to blocks.